src/BooleanSlider/widget/BooleanSlider.js

```diff
@@ -117,8 +117,8 @@ define([
             if (this._contextObj !== null) {
                 dojoStyle.set(this.domNode, "display", "block");
 
-                dojoProp.set(this.trueNode, "textContent", this.trueValue);
-                dojoProp.set(this.falseNode, "textContent", this.falseValue);
```

Comment:
Keep up the good work! Thanks for the pull request 😃 , it's good that you are working on widgets. And yes, they can be a pain in the ... to maintain. If there is anything we can assist you with, feel free to ask